build.gradle

```diff
@@ -1,14 +1,20 @@
 buildscript {
+    ext.kotlin_version = '1.4.0'
     apply from: rootProject.file('gradle/dependencies.gradle')
 
     repositories {
         jcenter()
     }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
```

Comment:
I'm pretty sure this isn't needed as the `plugins` `id` style takes care of it.

Reply:
Removed after updating `plugins` to `1.4.0`